custom/dns_interceptor/go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/litmuschaos/dns_interceptor
 
-go 1.19
+go 1.20
```

Comment:
It has cgo dependency - we can test the binary once

Reply:
Tested - 

```
vedant_shrotria@qa-lci-vm-2:~/test-tools/custom/dns_interceptor$ go mod tidy
go: downloading github.com/miekg/dns v1.1.41
go: downloading github.com/sirupsen/logrus v1.9.3
go: downloading golang.org/x/sys v0.5.0
go: downloading github.com/stretchr/testify v1.7.0
go: downloading golang.org/x/net v0.7.0
go: downloading golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/pmezard/go-difflib v1.0.0
go: downloading gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c
vedant_shrotria@qa-lci-vm-2:~/test-tools/custom/dns_interceptor$ go build
vedant_shrotria@qa-lci-vm-2:~/test-tools/custom/dns_interceptor$ ls
README.md  dns_interceptor  go.mod  go.sum  main.go  server
vedant_shrotria@qa-lci-vm-2:~/test-tools/custom/dns_interceptor$ ./dns_interceptor
[INFO] No PID mentioned running in current ns
INFO[0000] DNS Interceptor Port                          port=53
INFO[0000] Upstream DNS Server                           server="127.0.0.53:53"
INFO[0000] Chaos Error Targets                           targets="[]"
INFO[0000] Chaos Spoof Map                               spoof_map="map[]"
FATA[0000] Failed to create Interceptor                  error="wrong chaos type for dns chaos"
vedant_shrotria@qa-lci-vm-2:~/test-tools/custom/dns_interceptor$ cat go.mod
module github.com/litmuschaos/dns_interceptor

go 1.20

require (
        github.com/miekg/dns v1.1.41
        github.com/sirupsen/logrus v1.9.3
)

require (
        golang.org/x/net v0.0.0-20220906165146-f3363e06e74c // indirect
        golang.org/x/sys v0.5.0 // indirect
)

replace golang.org/x/net => golang.org/x/net v0.7.0
vedant_shrotria@qa-lci-vm-2:~/test-tools/custom/dns_interceptor$ go version
go version go1.20.3 linux/amd64
```